src/apps/search-result/helpers.ts

```diff
@@ -9,8 +9,8 @@ export const getPageSizeFromDataAttributes = ({
   desktop,
   mobile
 }: PageSizeDataAttributes) => {
-  const { pageSize } = getDeviceConf("search", {
-    search: {
+  const { pageSize } = getDeviceConf("dplConfig", {
```

Comment:
I don't think dplConfig is the right configuration domain name. Maybe we could use `paginatedLists`?

Reply:
it also contains "coverTints"